src/main/java/com/nicusa/UiApplication.java

```diff
@@ -24,7 +29,50 @@
       model.put("content", "Hello World");
       return model;
     }
-    
+
```

Comment:
These @RequestMappings should be in their own controller classes and then loaded via a ComponentScan.